src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -136,6 +136,41 @@ public BooleanSupplier getOutsideLimit() {
         }
     }
 
+    public class ShooterMap {
```

Comment:
Please move `ShooterMap` to the `frc.robot.maps.subsystems` package.

Reply:
What does this mean?